src/hocon_schema.erl

```diff
@@ -767,7 +770,10 @@ do_plain_get([], Conf) ->
     %% value as-is
     Conf;
 do_plain_get([H | T], Conf) ->
-    Child = maps:get(H, Conf, undefined),
```

Comment:
Not this pr’s issue, but: we need another clause to handle when Conf is not a map.